src/view/com/composer/videos/SubtitleDialog.tsx

```diff
@@ -147,22 +151,24 @@ function SubtitleDialogInner({
                 />
               ))}
             </View>
+            {subtitleMissingLanguage && (
+              <Text style={[a.text_sm, t.atoms.text_contrast_medium]}>
+                Ensure you have selected a language for each subtitle file.
```

Comment:
Would it be possible to mark this string for localization? 🙏

Reply:
thanks!